datafusion/src/logical_plan/plan.rs

```diff
@@ -438,11 +439,11 @@ impl LogicalPlan {
                 }
                 true
             }
+            LogicalPlan::Explain { plan, .. } => plan.accept(visitor)?,
```

Comment:
This is the fix that enables LogicalPlan's `display_...` functions to show all info of the plan. The newly added tests capture this fix.